(javascript) update parameters and formatting of some references (#1912)
clerk/clerk-docs: docs/references/javascript/phone-number.mdx

### `toString()`

> Returns the in [E.164 format](https://en.wikipedia.org/wiki/E.164) which includes the country code and the phone number.. Can also be accessed via the `PhoneNumber.phoneNumber` attribute.

Author: ```suggestion
Returns the phone number value in [E.164 format](https://en.wikipedia.org/wiki/E.164). Can also be accessed via the `PhoneNumber.phoneNumber` attribute.
```
